ui/eureka/eureka.js

```diff
@@ -2056,8 +2056,12 @@ class EurekaTracker {
       let gRegex = this.Translate(this.options.Regex);
       let gFlagRegex = gRegex['gFlagRegex'];
       let match = log.match(gFlagRegex);
-      if (match)
+      if (match) {
+        // remove character name on Korean server (if not /echo)
```

Comment:
Hmm, it is surprising to me that there is different logic for different languages.  Is it possible to make them all behave the same?

Reply:
> Yeah, it's the same in English. In English at least, all the names are stripped from the flag text, so it doesn't get shown.

As you said before, it seems only Korean (or Chinese too) server is showing character name on map, so I've set only to work on Korean server.
This logic is removing string ahead of `:`, which always logged except /echo.
so, if this logic is also applied to English version, 
`some place name: <flag>` will be replaced to ` <flag>`.

Reply:
The English flag regex is:
`'gFlagRegex': Regexes.parse(/00:00..:(?:[^:]*:)?(.*)Eureka (?:Anemos|Pagos|Pyros|Hydatos) \( (\y{Float})\s*, (\y{Float}) \)(.*$)/),`

Yeah, I think you're correct that if somebody did `/say text: <flag>` then the `text` part would be removed.

Maybe `/00:00(?:38:|..:[^:]*:)(.*)Eureka ... etc etc` would remove the name from non-echoes but work for echo, and we could apply that to all languages?

Reply:
This is working!